DSGT-DLP/Deep-Learning-Playground
# modify github actions to push to aws

# Modify github actions to push training and django containers to AWS

Github Issue Number Here: #1136 
**What user problem are we solving?**
Following #1136 , we want to ensure that we can push the django docker container to the ECS fargate service `django`, and push the training docker container to the ECS service `training` with EC2 launch type

**What solution does this PR provide?**
I modified the Terraform plan and github actions slightly

**Testing Methodology**
1. From `dlp-terraform/ecs`, run `terraform apply`, ensuring the name of your AWS profile is set correctly in `main.tf`
2. Go into the AWS console to see what the alb URL is, and update the corresponding line in `next.config.js`
3. Start SST. From `serverless`, run `AWS_PROFILE=dlp pnpm run dev`
4. Run the "ECS Django Container Deployment" and "ECS Training Container Deployment", ensuring you select this branch
5. Run `ENVIRONMENT=production dlp-cli frontend start` locally. If the env var is set, it proxies requests to AWS
6. Try training a tabular model

https://github.com/DSGT-DLP/Deep-Learning-Playground/assets/47485510/8d507a74-8855-4a53-97af-f5c4666e91d8


**Any other considerations**
None
